plugins/scaffolder/src/api.ts

```diff
@@ -75,6 +82,13 @@ export interface ScaffolderApi {
   // Returns a list of all installed actions.
   listActions(): Promise<ListActionsResponse>;
 
+  // Register Custom Fields
+  registerCustomField(field: CustomField): void;
+  getCustomFields(): {
```

Comment:
Why not return `Map<string, CustomField>`? :D

Reply:
I did this late night ok? :joy: